man/oci-image-tool-validate.1.md

```diff
@@ -19,6 +19,7 @@ oci-image-tool validate \- Validate one or more image files
   The reference to validate (should point to a manifest).
   Can be specified multiple times to validate multiple references.
   `NAME` must be present in the `refs` subdirectory of the image.
+  Format is A=B. Only support 'name', 'platform.os' and 'digest' three cases.
```

Comment:
Does it still work if user specify multiple ref? I haven't tested this, please test this.

Reply:
```
➜  ~ oci-image-tool validate --ref platform.os=linux --ref name=latest busybox
oci-image-tool: reference "platform.os=linux": OK
oci-image-tool: reference "name=latest": OK
busybox: OK
Validation succeeded
```

It's ok.